Heatington.Console/ConsoleUI.cs

```diff
@@ -93,7 +93,7 @@ private void LoadData()
         _dataPoints = sdm.TimeSeriesData!;
 
         opt.LoadData();
-        opt.OptimizeScenario1();
+        opt.Optimize();
```

Comment:
I don't see any optimize method from the optimizer. I guess, that OPT is planned to have only 1 main optimize method then?